app/views/idv/shared/_verify.html.erb

```diff
@@ -4,84 +4,94 @@ locals:
+          <dd class="display-inline margin-left-05"> <%= pii[:state_id_number] %> </dd>
         </div>
       <% end %>
-    </div>
+    </dl>
     <% if !remote_identity_proofing %>
       <div class='grid-auto'>
         <%= button_to(
               step_url.call(step: :redo_state_id),
               method: :put,
               class: 'usa-button usa-button--unstyled',
-              'aria-label': t('doc_auth.buttons.change_state_id_label'),
-            ) { t('doc_auth.buttons.change_label') } %>
+              'aria-label': t('idv.buttons.change_state_id_label'),
+            ) { t('idv.buttons.change_label') } %>
       </div>
     <% end %>
   </div>
   <div class="grid-row grid-gap grid-gap-2 padding-bottom-1 padding-top-1 border-y border-primary-light">
-    <div class='grid-col-fill'>
+    <dl class='grid-col-fill margin-bottom-0'>
       <div>
-        <%= "#{t('doc_auth.forms.address1')}: #{pii[:address1]}" %>
+        <dt class="display-inline"> <%= t('idv.form.address1') %>: </dt>
+        <dd class="display-inline margin-left-05"> <%= pii[:address1] %> </dd>
       </div>
       <div>
-        <%= "#{t('doc_auth.forms.city')}: #{pii[:city]}" %>
+        <dt class="display-inline"> <%= t('idv.form.city') %>: </dt>
+        <dd class="display-inline margin-left-05"> <%= pii[:city] %> </dd>
       </div>
       <div>
-        <%= "#{t('doc_auth.forms.state')}: #{pii[:state]}" %>
+        <dt class="display-inline"> <%= t('idv.form.state') %>: </dt>
+        <dd class="display-inline margin-left-05"> <%= pii[:state] %> </dd>
       </div>
       <div>
-        <%= "#{t('doc_auth.forms.zip_code')}: #{pii[:zipcode]}" %>
+        <dt class="display-inline"> <%= t('idv.form.zipcode') %>: </dt>
+        <dd class="display-inline margin-left-05"> <%= pii[:zipcode] %> </dd>
       </div>
-    </div>
+    </dl>
     <div class='grid-auto'>
       <%= button_to(
             step_url.call(step: :redo_address),
             method: :put,
             class: 'usa-button usa-button--unstyled',
-            'aria-label': t('doc_auth.buttons.change_address_label'),
-          ) { t('doc_auth.buttons.change_label') } %>
+            'aria-label': t('idv.buttons.change_address_label'),
+          ) { t('idv.buttons.change_label') } %>
     </div>
   </div>
   <div class="grid-row grid-gap grid-gap-2 padding-top-1">
     <div class='grid-col-fill'>
-      <%= t('doc_auth.forms.ssn') %>:
-      <%= render(
-            'shared/masked_text',
-            text: SsnFormatter.format(pii[:ssn]),
-            masked_text: SsnFormatter.format_masked(pii[:ssn]),
-            accessible_masked_text: t(
-              'idv.accessible_labels.masked_ssn',
-              first_number: pii[:ssn][0],
-              last_number: pii[:ssn][-1],
-            ),
-            toggle_label: t('forms.ssn.show'),
-          ) %>
+      <%= t('idv.form.ssn') %>: 
+      <div class="display-inline margin-left-05">
```

Comment:
I had to add a 0.5 left margin to match the styling of the description lists above

Reply:
nvm this isn't actually necessary -- removed in 9abec0194